packages/riipen-ui/src/components/Tab.jsx

```diff
@@ -104,7 +107,14 @@ class Tab extends React.Component {
 
     return (
       <React.Fragment>
-        <div className={className} onClick={this.handleClick}>
+        <div
+          tabIndex="0"
```

Comment:
Is this correct? Would this always put it as the first tab location even if it's in the middle of the page?

Reply:
took a bit of time looking into how this works yesterday actually

tabIndex -1: The item can be focused but must be done by a script, using the tab key wont work
tabIndex 0: The item can be focused by script or tab key
Any item with the same tabIndex (default is 0 for most things that can be focused) will tab in order from top to bottom
If an item has a higher tab index it follows the above rule but with higher priority.
So you could have everything on your page have a tab index of 1 and focus would work the same as it does now. 

Hopefully I explained it well @jordanell

Reply:
oh idea how to explain, it works in the same way as zIndex